examples/detailedlist/pyproject.toml

```diff
@@ -35,10 +35,12 @@ requires = [
 # Mobile deployments
 [tool.briefcase.app.detailedlist.iOS]
 requires = [
-    'toga-iOS',
+'../../src/core',
+'../../src/ios',
 ]
 
 [tool.briefcase.app.detailedlist.android]
 requires = [
-    'toga-android',
+'../../src/core',
+'../../src/android',
```

Comment:
These should be restored to their original state for consistency. That said... there might be an argument to be made for changing the demos to point at the local code implementation...